TEMPLATE/_mapdata/init.sls

```diff
@@ -2,18 +2,23 @@
 # vim: ft=sls
 ---
 {#- Get the `tplroot` from `tpldir` #}
-{%- set tplroot = tpldir.split('/')[0] %}
-{%- from tplroot ~ "/map.jinja" import TEMPLATE as mapdata with context %}
+{%- set tplroot = tpldir.split("/")[0] %}
+{%- from tplroot ~ "/map.jinja" import TEMPLATE with context %}
 
-{%- do salt['log.debug']('### MAP.JINJA DUMP ###\n' ~ mapdata | yaml(False)) %}
+{%- set _mapdata = {
+      "values": {
+        "TEMPLATE": TEMPLATE,
+      }
+    } %}
```

Comment:
I completely missunderstood your intent, I thought it will be:

```jinja
{%- set _mapdata = {
      "values": TEMPLATE,
    } %}
```

Because now, the output is:

``` yaml
values:
  TEMPLATE:
    added_in_defaults: defaults_value
```

instead of `defaults.yaml`:

``` yaml
values:
  added_in_defaults: defaults_value
```

:-/

Reply:
Ok, now I saw the code in [ssf-formula](https://github.com/myii/ssf-formula/blob/0965c571454a788bea59ab6354ab6bffe1a9eb88/ssf/files/default/formula/_mapdata/init.sls) I understand that it's related to version of `map.jinja`.

Sorry for the noise.